src/main/knative/deploy.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: cloudevents-player
 data:
   application.properties: |
-    brokerUrl/mp-rest/url=http://default-broker.default.svc.cluster.local
+    com.redhat.syseng.tools.cloudevents.service.BrokerService/mp-rest/url=http://default-broker
```

Comment:
The key should be fine as it is. I have to investigate more about this error.
See https://github.com/ruromero/cloudevent-player/blob/master/src/main/java/com/redhat/syseng/tools/cloudevents/service/BrokerService.java#L17

Reply:
That's what I thought, but I ended up with errors when applying the current `deploy.yaml` in a cluster.